src/styles/index.scss

```diff
@@ -0,0 +1,72 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,700&family=Poppins:wght@300&display=swap');
+
+$button-background: #fff;
+$button-font-color: #8f8f8f;
+$button-darker-font-color: #555;
+$dark-blue: #090930;
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background-color: $dark-blue;
+}
+
+#header {
+  font-weight: 700;
+  color: $dark-blue;
+}
+
+.task--wrapper {
+  padding: 0;
+
+  @media (min-width: 768px) {
+    width: 60%;
+  }
+}
+
+input[type='text'] {
+  outline: none;
+
+  &::placeholder {
+    font-style: italic;
+  }
+}
+
+input[type='checkbox'] {
+  transition: all 0.2s ease-in-out;
+
+  &:checked {
+    accent-color: #2ecd2e;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+}
```

Comment:
Great work with the styling and UX effects! I will just suggest applying a minor change during the edition moment to improve the user experience. Just to let the user understand that at that moment, it is in edition mode.  Maybe change it to italic, increase the font, or change the color. At this moment, my instinct makes me review it twice before editing, if it is available.